pkg/paravirt/paravirt_protected_entity_type_manager.go

```diff
@@ -323,19 +323,20 @@ func (this *ParaVirtProtectedEntityTypeManager) getSuperPVCandGuestPVCName(metad
 	svcPVC := v1.PersistentVolumeClaim{}
 	err := svcPVC.Unmarshal(metadata)
 	if err != nil {
-		return nil, "", "", errors.Wrapf(err, "failed to unmarshal metadata to get PVC")
+		return nil, "", "", map[string]string{}, errors.Wrapf(err, "failed to unmarshal metadata to get PVC")
 	}
 	// Construct Supervisor Cluster PVC name
 	pvcUUID, err := uuid.NewRandom()
 	if err != nil {
 		// NOTE: svcPVC is marshaled from metadata which is originally
 		// from the Guest Cluster PVC
-		return nil, "", "", errors.Wrapf(err, "could not generate PVC name in the Supervisor Cluster for Guest Cluster PVC %s/%s",
+		return nil, "", "", map[string]string{}, errors.Wrapf(err, "could not generate PVC name in the Supervisor Cluster for Guest Cluster PVC %s/%s",
 			svcPVC.Name, svcPVC.Namespace)
 	}
 	// Save original Guest Cluster PVC name and namespace
 	gcPVCName := svcPVC.Name
 	gcPVCNamespace := svcPVC.Namespace
+	gcPVCLabel := svcPVC.Labels
```

Comment:
vSphere CSI Driver does not keep Labels on Supervisor PVC.  We should do the same during restore.
Add a line after this to clear out Labels from svcPVC and add comments to the change.

Reply:
Clear labels from svcPVC and updated documentation.